### PR TITLE
[Rewards 3.0] Add payout account card

### PR DESCRIPTION
**Pull request by zenparsing**
Resolves https://github.com/brave/brave-browser/issues/40504
Resolves https://github.com/brave/brave-browser/issues/40428

Screenshot 2024-08-16 at 3 39 41 PM
Screenshot 2024-08-16 at 3 40 30 PM

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: components/brave_rewards/resources/rewards_page/lib/locale_strings.ts [hunk 1]
```diff
@@ -115,6 +115,16 @@ export const localeStrings = {
   onboardingTextItem1: 'Get paid for private ads you see in Brave',
   onboardingTextItem2: 'Get special benefits and discounts',
   onboardingTitle: 'Turn on Brave Rewards',
+  payoutAccountBalanceLabel: 'Your balance',
+  payoutAccountConnectedLabel: 'Connected',
+  payoutAccountDetailsTitle: '$1 wallet',
+  payoutAccountLabel: 'Payout account',
+  payoutAccountLink: 'Go to my  account',
```
Comment (emerick) [inline components/brave_rewards/resources/rewards_page/lib/locale_strings.ts:122]: Nit: This entry has an extra space in it :-)

**Review by emerick — APPROVED** [review]

File: components/resources/rewards_strings.grdp [hunk 2]
```diff
@@ -787,4 +787,35 @@
...
+    Connected
+  </message>
+  <message name="IDS_REWARDS_PAYOUT_ACCOUNT_DETAILS_TITLE" desc="">
+    <ph name="PROVIDER">$1</ph> wallet
+  </message>
+  <message name="IDS_REWARDS_PAYOUT_ACCOUNT_LABEL" desc="">
+    Payout account
+  </message>
+  <message name="IDS_REWARDS_PAYOUT_ACCOUNT_LINK" desc="">
+    Go to my  account
```
[the remark this answers is not in the document]
Comment (zenparsing) [reply components/resources/rewards_strings.grdp]: Fixed

File: components/brave_rewards/resources/rewards_page/lib/locale_strings.ts (hunk 1 above)
Comment (zenparsing) [reply components/brave_rewards/resources/rewards_page/lib/locale_strings.ts]: Fixed

**Review by mkarolin — APPROVED** [review]
`strings`++

**Comment by Miyayes** [comment]
image

This should say "Log in" instead of "Login" in both cases, since it's the verb form

**Comment by zenparsing** [comment]
@Miyayes I merged before seeing your comment. I'll get those strings updated in the next PR.

**Pull request merged**